schema/defs/vrs/Haplotype.rst

```diff
@@ -26,5 +26,5 @@ Some Haplotype attributes are inherited from :ref:`Variation`.
       - MUST be "Haplotype"
    *  - members
       - :ref:`Allele` | :ref:`CURIE`
-      - 1..m
+      - 2..m
```

Comment:
@ahwagner is this a breaking change? Or are we not worrying about any breaking changes until we establish the maturity model? 

We may want to call out that this does invalidate previous versions that had haplotypes of 1 implemented.  I'm not sure how to do this, but we should consider annotating these types of "minor" changes that are borderline or actual breaking changes that we are including because we feel as if the model is in an Draft or Trial Use state.

Reply:
This doesn't change digests of valid Haplotypes–only reduces the number of Haplotypes that are considered valid. I see this as the sort of thing that can be easily remedied in software–Haplotypes with 1 member will just be transformed into the member Allele.

We are avoiding any changes in 1.x that alter digests (though we have deprecated / replaced Allele properties already with SequenceExpressions), but I think anything else is fair game.